src/scripts/menus/help.js

```diff
@@ -1,21 +1,30 @@
 import { remote } from 'electron';
+import servers from '../servers';
 const APP_NAME = remote.app.getName();
 
 const helpTemplate = [
     {
         label: APP_NAME + ' Help',
-        click: function () {
-            remote.shell.openExternal('https://rocket.chat/docs');
-        }
+        click: () => remote.shell.openExternal('https://rocket.chat/docs')
+    },
+    {
+        type: 'separator'
+    },
+    {
+        label: 'Report Issue',
```

Comment:
I really like this idea!  We get a lot of issues that we have to shift around to the different repos.  I wonder if something like this would help